### DISCUSSION TITLE
Omit Achievements with 0 Retro Ratio from Easy Achievements List

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#784] [Ideas]
https://retroachievements.org/achievementList.php?s=4&p=2

The first 85+ pages (roughly 2200 achievements) are all achievements which have not been earned by anyone, making this list very inaccurate.

**Comment by juherr** [comment]
I don't know how you found the url but it looks it is working as expected:

The default order of https://retroachievements.org/achievementList.php is "by added date".
You're URL is asking for an order "by ratio asc".
If you want "by ratio desc", just use https://retroachievements.org/achievementList.php?s=14&p=2

**Comment by televandalist** [comment]
I know. The point was to remove games with a 0 retro ratio (meaning they've never been earned) from the easy achievement list. The first 85 pages were unearned achievements, which is inaccurate. 

The URL I posted is the link for "Easy Achievements" near the top of the Achievements dropdown.

**Comment by juherr** [comment]
Ok, I see.

Do you know why "easy" is https://retroachievements.org/achievementList.php?s=4&p=2 and "hardest" is https://retroachievements.org/gameSearch.php?p=0 ?
What is the definition of an easy achievement vs a hard achievement?

**Comment by Jamiras** [comment]
I believe this was addressed in #941
